src/registrar/models/domain_information.py

```diff
@@ -177,6 +180,7 @@ class DomainInformation(TimeStampedModel):
         null=True,
         # Access this information via Domain as "domain.domain_info"
         related_name="domain_info",
+        verbose_name="domain request",
```

Comment:
**Question:** Why is the verbose name domain request versus domain info 😅

Reply:
😅 Also an error on my part - just removed this and thank you for catching this!